src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Models/GatewayInfo.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.Analysis.Models;
+
+namespace Microsoft.Azure.Commands.AnalysisServices.Models
```

Comment:
Please add "PS" prefix to model name to distinguish between the PS type and the SDK type.